UGW: export of DESCRIPTION as defined in package-ipkg.mk is conflicting in case of bootcore
	 as bootcore model is compiled as package under main model.
	 Renaming the DESCRIPTION flag in package-ipkg.mk to PKG_DESCRIPTION will solve the issue.

diff --git a/include/package-ipkg.mk b/include/package-ipkg.mk
--- a/include/package-ipkg.mk
+++ b/include/package-ipkg.mk
@@ -186,7 +186,7 @@
 
     $(PKG_INFO_DIR)/$(1).provides: $$(IPKG_$(1))
     $$(IPKG_$(1)) : export CONTROL=$$(Package/$(1)/CONTROL)
-    $$(IPKG_$(1)) : export DESCRIPTION=$$(Package/$(1)/description)
+    $$(IPKG_$(1)) : export PKG_DESCRIPTION=$$(Package/$(1)/description)
     $$(IPKG_$(1)): $(STAMP_BUILT) $(INCLUDE_DIR)/package-ipkg.mk
 	@rm -rf $$(PDIR_$(1))/$(1)_* $$(IDIR_$(1))
 	mkdir -p $(PACKAGE_DIR) $$(IDIR_$(1))/CONTROL $(PKG_INFO_DIR)
@@ -207,7 +207,7 @@
 	(cd $$(IDIR_$(1))/CONTROL; \
 		( \
 			echo "$$$$CONTROL"; \
-			printf "Description: "; echo "$$$$DESCRIPTION" | sed -e 's,^[[:space:]]*, ,g'; \
+			printf "Description: "; echo "$$$$PKG_DESCRIPTION" | sed -e 's,^[[:space:]]*, ,g'; \
 		) > control; \
 		chmod 644 control; \
 		( \
